src/main/kotlin/com/autonomousapps/tasks/ComputeUsagesTask.kt

```diff
@@ -132,11 +132,11 @@ private class GraphVisitor(
 
     var capabilities = dependency.capabilities
 
-    dependencyCoordinates.gradleVariantIdentification.externalVariant?.let { externalVariant ->
-      // If this is just a shim to an external variant, defer to its capabilities instead
-      val externalIdentifier = externalVariant.capabilities.single()
-      context.dependenciesByIdentifier[externalIdentifier]?.let { capabilities = it.capabilities }
-    }
+    // dependencyCoordinates.gradleVariantIdentification.externalVariant?.let { externalVariant ->
+    //   // If this is just a shim to an external variant, defer to its capabilities instead
+    //   val externalIdentifier = externalVariant.capabilities.single()
+    //   context.dependenciesByIdentifier[externalIdentifier]?.let { capabilities = it.capabilities }
+    // }
```

Comment:
I think I fundamentally agree with the earlier decision to do this. The "usages" should be _correct_ irrespective of what "makes sense" to developers. There must be a 1:1 correspondence between the dependencies and the usages of those dependencies in the consumer project, as an invariant. Much of the rest of the analysis relies on this.